app/src/main/res/drawable/ic_shortcut_my_courses.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="48dp" android:viewportHeight="240"
+    android:viewportWidth="240" android:width="48dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#F5F5F5" android:fillType="evenOdd" android:pathData="M120.1,119.9m-112,0a112,112 0,1 1,224 0a112,112 0,1 1,-224 0"/>
+    <path android:fillColor="@color/apptheme_primary" android:pathData="M95,61.7c-11.4,0 -24.3,8 -24.3,16.7v94c0,3 2.7,5.7 5.7,5.7h71.1c3,0 5.7,-2.7 5.7,-5.7V84.1c0,-3 -2.7,-6.5 -6.5,-6.5H83.2c-4.2,0 4.2,-8 11,-8h63.5c2.7,0 4.2,1.5 4.2,4.2v94.3c0,4.6 8,1.1 8,-4.2V71.2c0,-4.9 -4.6,-9.5 -10.7,-9.5H95L95,61.7zM111.3,83.7h23.2v63.5l-11.8,-11l-11.4,11.4C111.3,147.6 111.3,83.7 111.3,83.7z"/>
+</vector>
```

Comment:
Please check the formatting of our other vector drawables. I'm quite sure that Reformat Code will do that for you.